### DISCUSSION TITLE
希望支持微软azure部署的OpenAI服务

### DISCUSSION THREAD

**Discussion by xingfly** [immersive-translate/immersive-translate#684] [Ideas]
### 新功能 | New Feature

1. 希望支持微软azure部署的OpenAI服务

### 为什么您想要添加这些功能 | Why you want to add these features

希望支持微软azure部署的OpenAI服务

### 提议 | Proposal

希望支持微软azure部署的OpenAI服务

**Comment by theowenyoung** [comment]
已经支持的哈 https://immersivetranslate.com/docs/services/openai/#azure-openai
